mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -1,30 +1,59 @@
 CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManager
+	COMMENT A manager of dynamic registries. It allows users to access non-hardcoded
+	COMMENT registries reliably.
+	COMMENT
+	COMMENT <p>Each minecraft server has a dynamic registry manager for file-loaded
+	COMMENT registries, while each client play network handler has a dynamic registry
+	COMMENT manager for server-sent dynamic registries.</p>
+	COMMENT
+	COMMENT <p>The {@link net.minecraft.util.registry.DynamicRegistryManager.Impl}
+	COMMENT class serves as an immutable implementation of any particular collection
+	COMMENT or configuration of dynamic registries.</p>
 	FIELD field_25918 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_25919 INFOS Ljava/util/Map;
 	METHOD method_30518 getDimensionTypes ()Lnet/minecraft/class_2378;
 	METHOD method_30519 load (Lnet/minecraft/class_3300;)Lnet/minecraft/class_5455$class_5457;
+		COMMENT Loads a dynamic registry manager from the resource manager's data files.
 		ARG 0 resourceManager
 	METHOD method_30520 register (Lcom/google/common/collect/ImmutableMap$Builder;Lnet/minecraft/class_5321;Lcom/mojang/serialization/MapCodec;Z)Lcom/google/common/collect/ImmutableMap$Builder;
+		COMMENT Registers a registry handling logic to the immutable map builder for
```

Comment:
```suggestion
		COMMENT Adds a {@linkplain DynamicRegistryManager.Info} to the immutable map builder for
```

Reply:
an info is a registry handling logic. i'd say registry handling logic is more clear.

Reply:
What do you mean by "handling"? It's also bad grammar because "logic" is not a count noun.

Reply:
The Info class is a logic to handle a particular registry's serialization behavior.

Reply:
Yes but:
- nowhere else do you refer to refer to the `Info` class as "registry handling logic"
- "registry serializing logic" is better
- if that's what the `Info` class is, why not rename it from `Info` to `SerializingLogic`?
- your grammar is still wrong.